core/scanners.py

```diff
@@ -0,0 +1,128 @@
+"""MystBin. Share code easily.
+
+Copyright (C) 2020-Current PythonistaGuild
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import base64
+import binascii
+import enum
+import logging
+import re
+from typing import ClassVar
+
+from types_.scanner import ScannerSecret
```

Comment:
Any imports from the specific `types_` or any typing only spaces must be within a TYPE_CHECKING block due to runtime access errors (or the potential of).